docs/features/profiles/marriages/marrying.md

```diff
@@ -2,20 +2,20 @@
 
 ## How to propose to a user
 
-Use the `-marry @mention` command.
+Use the `/marry propose <user>` command.
 
 ## Accepting a marriage proposal
 
-Use `-marry accept @mention` to accept the person's marriage proposal.
+Use `/marry accept <fiancé>` to accept the person's marriage proposal.
 
 ## Canceling a marriage proposal
 
-To cancel a marriage proposal you have made to someone, use `-marry cancel @mention`.
+To cancel a marriage proposal you have made to someone, use `/marry cancel <fiancé>`.
```

Comment:
Here `fiancé` is optional so `[fiancé]`. Running it without the fiancé will give you a dropdown like for divorcing someone off-server.